src/app/unlocked-stage/unlocked-stage.component.html

```diff
@@ -1,20 +1,21 @@
 <div id="container" [hidden]="gameOver || success">
     <div id="scene">
-        <div id="human-health-text">{{language?.legend.human}}</div>
+      <div class="progress  progress-bar-vertical">
```

Comment:
spacing..

Reply:
the spacing here is still a problem..

Reply:
@lunayach  this still needs to be solved..

Reply:
If you meant to align with the adjacent DIV's. I changed it.